api/rql/internal/primary/timeAttr_test.go

```diff
@@ -82,5 +72,5 @@ func (s *TimeAttrTestSuite) TestExpression_AtomAndNot() {
 	s.setAttr(&e, s.TM(500))
 	s.EEFTC(expr, e)
 
-	s.EESTTC(expr, schema)
+	//s.EESTTC(expr, schema)
```

Comment:
Commented?

Reply:
Ah yeah, I wanted to hold off deleting this until NPE/PE was resolved. Since NPE's are OK, I'll remove this commented stuff.

Reply:
Updated.